rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -612,13 +616,6 @@ def decode_transaction_hashes(
         refresh_balances = False
         with self.database.conn.read_ctx() as cursor:
             self.reload_data(cursor)
-            # If no transaction hashes are passed, decode all transactions.
```

Comment:
Hmmm you removed this functionality. It was not used anywhere apart from that test you deleted?

Reply:
Yes, not used anywhere else